Update for React Native tvOS (#128)
mParticle/docs: src/pages/developers/sdk/react-native/getting-started/index.md

$ npm install react-native-mparticle --save
```

> For clients using the [react-native-tvos library](https://github.com/react-native-tvos/react-native-tvos) you will need to add the `force` flag to the install code as shown below. This tells the installer to ignore using the core react native repository and instead use the tvOS specific library instead.

Reviewer: ```suggestion
For clients using the [react-native-tvos library](https://github.com/react-native-tvos/react-native-tvos) you must add the `force` flag to the install code as shown below. This tells the installer to ignore using the core React Native repository and instead use the tvOS-specific library instead.
```

> For clients using the [react-native-tvos library](https://github.com/react-native-tvos/react-native-tvos) you will need to add the force flag to to the install code shown bellow. This allows the install to ignore that you are not using the core react native repository and use the tvos specific library instead.

Reviewer 2: ```suggestion
For clients using the [react-native-tvos library](https://github.com/react-native-tvos/react-native-tvos) you will need to add the `force` flag to the install code as shown below. This tells the installer to ignore using the core react native repository and instead use the tvOS specific library instead.
```
